docker-entrypoint.sh

```diff
@@ -49,11 +49,6 @@ for VAR_NAME in $(env | grep '^GRAYLOG_[^=]\+__FILE=.\+' | sed -r 's/^(GRAYLOG_[
 done
 
 
-
-if [ "${1:0:1}" = '-' ]; then
-  set -- graylog "$@"
-fi
-
```

Comment:
If the first argument starts with a dash, it adds `graylog` in front of the arguments.
I don't know what this was for, but we don't have a graylog command, so I assume that this must be quite old and can be removed.

Reply:
Correction: this probably was referring to the `graylog()` shell function. but whoever introduced `run()` made that obsolete

Reply:
It was an indicator to start the Graylog server instead of executing an arbitrary command. It becomes apparent when you look at the initial version of the script: https://github.com/Graylog2/graylog-docker/commit/001378bf96093802f2dce5842923fcbcb2e557ee#diff-79738685a656fe6b25061bb14181442210b599f746faeaba408a2401de45038a

As Marco said, it doesn't seem used anymore.

Reply:
Okay, great. Thanks for the clarification.